new_core_concepts/modules/ROOT/pages/typeql/query-variables-patterns.adoc

```diff
@@ -77,10 +76,10 @@ We will continue writing out the constraints for illustrative purposes.
 What does it mean to [[definition-match]]"*match*" a pattern?
 A conjunctive pattern is "satisfied" if each variable can be mapped to some concept
 so that every constraint in the pattern holds true in the database,
-and every sub-pattern is also satisfied.
+and every subpattern is also satisfied.
 
 Each such mapping from variables to concepts is an [[definition-answer]]*answer* to the pattern.
-For the query above, each answer maps `$e` to an entity type, and `$a` to an attribute type owned by that type.
+For the query above, each answer maps `$e` to an entity, and `$a` to an attribute owned by that type.
```

Comment:
I might actually prefer to use entity *type* and attribute *type* here.
I just avoided it in the first section because it was very introductory.